sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -538,12 +539,25 @@ case class TruncateTableCommand(
               }
             }
             optAcls.foreach { acls =>
+              val aclEntries = acls.asScala.filter(_.getName != null).asJava
+
+              // The ACL API also expects the tradition user/group/other permission
+              // in the form of ACL.
```

Comment:
Could you revise a comment to mention that this happens only when the default ACL configuration doesn't exist?

Reply:
Yea, sure. Let me revise it.